package.json

```diff
@@ -80,7 +80,7 @@
     "axios": "^0.19.0",
     "babel-core": "6",
     "babel-loader": "7",
-    "babel-plugin-istanbul": "^5.2.0",
+    "babel-plugin-istanbul": "4.1.6",
```

Comment:
Apparently `test:coverage` has been broken for while since the incompatibility on bable
- siw using babel 6
- plugin-istanbul 5 is using babel 7

Hence rollback to old version.